gbeam_library/Vertex[] vertices_obstacles
gbeam_library/Vertex[] vertices_reachable
